packages/slate-editor/src/extensions/coverage/components/CoverageElement.tsx

```diff
@@ -60,6 +60,16 @@ export function CoverageElement({
         }
     }
 
+    function handleCoverageUpdated({ coverage }: { coverage: Pick<CoverageEntry, 'id'>}) {
+        if (element.coverage.id === coverage.id) {
+            loadCoverage();
+        }
+    }
+
+    useEffect(() => {
+        EventsEditor.addEventListener(editor, 'coverage-updated', handleCoverageUpdated);
+    }, []);
```

Comment:
1. Shouldn't `handleCoverageUpdated` be a local function to the effect callback?
2. Should it not remove the event listener in the effect unmount callback?

Reply:
You're right, I forgot 🙂